# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/wchar.h.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The wchar.h mostly has prototypes that are not implemented. Configure
might succeed, but link will fail.

  - Rene Rebe <rene@exactcod.de>

--- ./include/wchar.h.vanilla	2006-07-21 23:23:03.000000000 +0200
+++ ./include/wchar.h	2006-07-21 23:23:21.000000000 +0200
@@ -23,6 +23,8 @@
 #define WEOF 0xffffffffu
 #endif
 
+#if 0
+
 struct tm;
 
 typedef struct {
@@ -94,6 +96,8 @@
 int wprintf(const wchar_t *__restrict__, ...);
 int wscanf(const wchar_t *__restrict__, ...);
 
+#endif
+
 __END_DECLS
 
 #endif
